docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,476 @@
+- The SW upgrade has been enhanced to apply on more devices such as headless device:
+  - For small company that does not have SW upgrade infra, SW upgrade infra has been provided on basis of Samsung Smartthings DeviceManager (https://console.smartthingsdm.com). Now, it is available on Beta services.
+  - Remote update control for updating headless device has been provided. The OCF based update protocol and reference agents have been provided. In addition, remote update control framework has been provided to support several devices. The framework provides update control API set and update control plugins. The default plugin is available on the Tizen IoT homepage.
+  - For secure update, generic template for binary signing/verification has been provided. Default binary signer and verifier are available on the Tizen IoT homepage.
+- Low Memory Management for headless devices has been enhanced:
+  - Applications can be excluded from low memory killer by defining Out of Memory (OOM) exception in manifest file.
+  - OOM Score control interface has been provided for controlling active application like sound-focused application.
+- Swap system has been enhanced to support multiple swap types and devices.
+  - File-based swap and Zswap-based swap has been added.
+  - Early memory reclaiming (to swap) feature has been added.
+
+### Fixes
+
+- The bug that does not deliver low memory notification through event system has been fixed for headless devices.
+
+
+## System (Base)
+
+### New and changed features
+
+- Upgraded the following open source libraries:
+  - icu (60.2)
+  - ncurses (6.1)
+    - CVE-2018-10754, CVE-2017-16879, CVE-2017-13734, CVE-2017-13733, CVE-2017-13732, CVE-2017-13731, CVE-2017-13730, CVE-2017-13729, CVE-2017-13728
+  - sqlite (3.24.0)
+    - CVE-2018-8740
+  - procps-ng (3.3.15)
+    - CVE-2018-1122, CVE-2018-1123, CVE-2018-1124, CVE-2018-1125, CVE-2018-1126
+  - unzip (6.10c23)
+    - CVE-2018-1000031, CVE-2018-1000032, CVE-2018-1000033, CVE-2018-1000035
+  - json-glib (1.4.2)
+- Expanded the capi-base-utils API set: 
+  - Simple Date Format
+  - Locale Display Names
+
+
+## Application framework
+
+### New and changed features
+
+- Tizen Interface Definition Language (TIDL), rpc-port
+  - rpc-port is now available in the wearable profile.
+  - TIDL has been improved in terms of performance and stability.
+- Watchface complication framework
+  - API has been refined to support more complex UX.
+- App control API
+  - A new API for sending launch request asynchronously has been added.
+- Minicontrol API
+  - Minicontrol API is now available in the wearable profile.
+
+
+## Window System
+
+### New and changed features
+
+- Wayland
+  - The open source wayland has been upgraded to the 1.15.0 version.
+- Extended Wayland Protocols
+  - tizen_move_resize protocol has been added to support move and resize wl_surface at the same time by passing x,y,w,h geometry parameters.

```

Comment:
Can we rephrase as:
tizen_move_resize protocol has been added to support moving and resizing wl_surface at the same time by passing x,y,w,h geometry parameters.

Reply:
Your suggestion is fine. :)

Reply:
Thank you.